tasks/client_config.yaml

```
@@ -1,10 +1,33 @@
 ---
+- name: Check for root password
+  changed_when: false
+  stat:
+    path: /usr/local/etc/mysql-password
+  register: mariadb_mysql_password
+
+- name: create root password
+  when: mariadb_mysql_password.stat.exists == false
+  copy:
+    dest: /usr/local/etc/mysql-password
+    content: "{{ lookup('password', '/tmp/passwordfile length=45 chars=ascii_letters,digits') }}"
+    owner: "{{ system.user.root.name }}"
+    group: "{{ system.user.root.group }}"
+    mode: '0600'
+
 - name: Read root-password
   slurp:
     src: "{{ item }}"
   with_items:
     - /usr/local/etc/mysql-password
-  register: slurp_result
+  register: mariadb_slurp_result
+
+- name: Set root password in database
+  when: mariadb.galera.cluster == false or mariadb.galera.initializer == true
```

Using `.__bool__()` instead of `.__eq__()`. It's more pythonic (duck typing) and also shorter/easier to read. And the role and other roles already use this syntax.
```suggestion
  when: not mariadb.galera.cluster or mariadb.galera.initializer
```